### DISCUSSION TITLE
Reverse icons switch

### DISCUSSION THREAD

**Discussion by KodeXall** [jonaskohl/CapsLockIndicator#124] [Ideas]
I propose adding a simple option for switching around of icons - i.e. to use state ON as OFF and OFF as ON for a given key / icon or for all of them all together

The logic behind is that a simple / oldschool keyboard might have red light indicators for the ON states. And so a user might want to match them by using red colors for various elements of notifications - and thus a contradiction would then arise, because icons would display "opposite" color branding than notifications

**Comment by jonaskohl** [comment]
You could accomplish this by creating custom icons. Please refer to [this knowledgebase article](https://cli.jonaskohl.de/kb/?a=icons) for further details.

**Comment by KodeXall** [comment]
Yes, I could

But just as this program gives a simple way of accessing new sets of icons, so should it allow for a quick switch - especially considering that many of the already available icons packs utilize such red vs. green approach

**Comment by jonaskohl** [comment]
I'll consider this for a future update
